### PR TITLE
Publish to Chrome Webstore and Mozilla Addons

### PR DESCRIPTION
**Pull request by ErikBjare**
Solves https://github.com/SuperuserLabs/thankful/issues/30

Automatic deployment on tagged releases will be done in a later PR, see #33

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ node_js:
   - "8"
 
 install:
- - make install
+ - npm install
```
Comment (ErikBjare) [inline .travis.yml]: @Powersource Why did you change this? Changing it back.

File: Makefile [hunk 2]
```diff
@@ -23,3 +26,15 @@ typecheck:
...
+package:
+	cd dist/ && zip -r ../thankful.zip *
+
+publish:
+	# Use these to automatically publish
+	# https://www.npmjs.com/package/chrome-webstore-upload-cli
+	# https://www.npmjs.com/package/firefox-extension-deploy
+	node scripts/publish-mozilla-addons.sh
+	# Doing it like this would expose keys, not acceptable
+	#    webstore upload --source extension.zip --extension-id ${WEBSTORE_EXTENSION_ID} --client-id ${WEBSTORE_CLIENT_ID} --client-secret ${WEBSTORE_CLIENT_SECRET} --refresh-token ${REFRESH_TOKEN}
```
Comment (Powersource) [inline Makefile]: `Webstore` by default gets these from envvars, you don't have to specify it.

And shouldn't we be safe by putting them as secret envvars in travis?

File: Makefile [hunk 3]
```diff
@@ -23,3 +26,15 @@ typecheck:
...
 	rm -v dist/**/*.{js,html}
+
+package:
+	cd dist/ && zip -r ../thankful.zip *
+
+publish:
+	# Use these to automatically publish
+	# https://www.npmjs.com/package/chrome-webstore-upload-cli
+	# https://www.npmjs.com/package/firefox-extension-deploy
+	node scripts/publish-mozilla-addons.sh
```
Comment (ahnlabb) [inline Makefile]: `.sh`? Should be `.js` right?

File: Makefile (hunk 2 above)
Comment (ErikBjare) [reply Makefile]: Yeah, the only issue is that we'd want to name them something different in the envvars as they might otherwise conflict.

File: Makefile (hunk 3 above)
Comment (ErikBjare) [reply Makefile]: Fixed!
